publish.html

```diff
@@ -652,11 +601,20 @@ <h2><a class="anchor" id="repository"></a>Using OMERO as a Public Repository</h2
 
 </ul>
 
+<p>
+    See the
+    <a href="https://www.openmicroscopy.org/explore/" target="_blank">Explore</a> page for more examples.
+</p>
+
 <div class="line-block">
 <div class="line"><br /></div> <!-- end #line -->
 </div> <!-- end #line-block -->
 
-<p>OMERO.gallery provides a basic gallery front end based on the Group-Project-Dataset-Image, and is a good starting point to develop a bespoke front end.</p>
+<p>
+    OMERO.gallery provides a basic gallery front end based on the
```

Comment:
Would be nice for OMERO.gallery to link to e.g. https://github.com/ome/omero-gallery